init.sql

```diff
@@ -0,0 +1,218 @@
+  `DateRanked` timestamp NULL DEFAULT NULL,
+  `Blacklisted` tinyint(1) NOT NULL DEFAULT '0',
+  `BlacklistReason` text COLLATE utf8mb3_bin,
+  PRIMARY KEY (`BeatmapID`),
+  KEY `beatmapset_id` (`SetID`),
+  FULLTEXT KEY `Artist` (`Artist`,`Title`,`DifficultyName`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_bin ROW_FORMAT=DYNAMIC;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `blacklist`
+--
+
+DROP TABLE IF EXISTS `blacklist`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `blacklist` (
+  `UserID` int NOT NULL,
+  PRIMARY KEY (`UserID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `comments`
+--
+
+DROP TABLE IF EXISTS `comments`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `comments` (
+  `CommentID` int NOT NULL AUTO_INCREMENT,
+  `UserID` int NOT NULL,
+  `SetID` int NOT NULL,
+  `Comment` text,
+  `date` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`CommentID`)
+) ENGINE=InnoDB AUTO_INCREMENT=10489 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `mappernames`
+--
+
+DROP TABLE IF EXISTS `mappernames`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `mappernames` (
+  `UserID` int DEFAULT NULL,
+  `Username` varchar(255) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `oldbeatmaps`
+--
+
+DROP TABLE IF EXISTS `oldbeatmaps`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `oldbeatmaps` (
+  `DateRanked` date NOT NULL,
+  `Artist` varchar(80) DEFAULT NULL,
+  `BeatmapID` int NOT NULL,
+  `SetID` int NOT NULL,
+  `CreatorID` int NOT NULL,
+  `SR` decimal(9,6) NOT NULL,
+  `CS` decimal(7,5) NOT NULL,
+  `OD` decimal(4,2) NOT NULL,
+  `AR` decimal(7,5) NOT NULL,
+  `HP` decimal(4,2) NOT NULL,
+  `Source` varchar(76) DEFAULT NULL,
+  `Genre` int NOT NULL,
+  `Lang` int NOT NULL,
+  `Title` varchar(80) DEFAULT NULL,
+  `DifficultyName` varchar(80) DEFAULT NULL,
+  `Mode` int NOT NULL,
+  `Rating` float DEFAULT NULL,
+  `ChartRank` int DEFAULT NULL,
+  `ChartYearRank` int DEFAULT NULL,
+  `Timestamp` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+  `RatingCount` int DEFAULT NULL,
+  `WeightedAvg` float DEFAULT NULL,
+  PRIMARY KEY (`BeatmapID`),
+  KEY `idx_setID` (`SetID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
```

Comment:
I forgot this old table was still here, we can probably do with removing this 😛 

```suggestion
```

Reply:
Fixed